scaladoc/src/dotty/tools/scaladoc/tasty/comments/markdown/SectionRenderingExtension.scala

```diff
@@ -33,7 +33,7 @@ object SectionRenderingExtension extends HtmlRenderer.HtmlRendererExtension:
       val Section(header, body) = node
       val idSuffix = repeatedIds.getOrElseUpdate((c, header.getText), 0)
       val ifSuffixStr = if(idSuffix == 0) then "" else idSuffix.toString
-      repeatedIds.update((c, header.getText), repeatedIds((c, header.getText)) + 1)
+      repeatedIds.update((c, header.getText), idSuffix + 1)
```

Comment:
Those the result of `header.getText` have a stable hash?

Reply:
I think it should, but just to be extra safe I extracted that in the next commit